### DISCUSSION TITLE
Delete local logs

### DISCUSSION THREAD

**Discussion by BillySchwartz16** [apache/airflow#41345] [Q&A]
Hi all,

On version 2.9.2. I am sending my logs to s3.  I set AIRFLOW__LOGGING__DELETE_LOCAL_LOGS = "True" in my airflow container, but the logs persist inside the container.  Why aren't they being deleted after upload like stated here: 

https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#delete-local-logs

Anyone have this working?

**Comment by RajasGujarathi** [comment]
We also observe the same issue with version 2.9.3

**Reply by BillySchwartz16** [reply]
We realized the log folders do not get deleted, but the actual log files are removed

**Comment by potiuk** [comment]
The description is - when you read the configuration:

```
Whether the local log files for GCS, S3, WASB and OSS remote logging should be deleted
after they are uploaded to the remote location.
```

So if you are SURE that you have this remote logging configured and it does not work - submit an issue with all the details of your configuration showing what happens and likely showing some logs from the uploading and then removal.

**Reply by RajasGujarathi** [reply]
I did not get the opportunity to conduct a detailed analysis. Will get back on this, as soon as I have the opportunity to do so.
